packages/jellyfish-api-core/src/index.ts

```diff
@@ -78,10 +78,10 @@ export class ClientApiError extends ApiError {
  * API RPC error, from upstream.
  */
 export class RpcApiError extends ApiError {
-  public readonly payload: { code: number, message: string }
+  public readonly payload: { code: number, message: string, rpcMethod?: string }
 
-  constructor (error: { code: number, message: string }) {
-    super(`RpcApiError: '${error.message}', code: ${error.code}`)
+  constructor (error: { code: number, message: string, rpcMethod?: string }) {
```

Comment:
Remove the optional aspect for `rpcMethod?: string` and add it to https://github.com/DeFiCh/jellyfish/blob/82b6692b48492bf109f95bbe3767f03bf32e0d7a/packages/jellyfish-api-jsonrpc/src/index.ts#L92-L94
as well